### PR TITLE
Added website relation

### PR DESCRIPTION
**Pull request by alesstimec**
The website relation will make it possible to relate the
controller charm to haproxy.

### CODE REVIEW

File: src/charm.py [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 import logging
+import os
+import yaml
 
+from charmhelpers.core import hookenv
 from ops.charm import CharmBase
 from ops.main import main
 from ops.model import ActiveStatus
 from ops.framework import StoredState
+from ops.model import BlockedStatus
```
Comment (wallyworld) [inline src/charm.py:14]: can we group the ops.model imports together, we have one above

File: tests/test_charm.py [hunk 2]
```diff
@@ -1,12 +1,48 @@
...
 
+import os
 import unittest
+from unittest.mock import patch, mock_open
 
 from ops.testing import Harness
 from charm import JujuControllerCharm
 
 
+agent_conf = '''
```
Comment (wallyworld) [inline tests/test_charm.py:12]: can we just set up the minimal key values needed for the test rather than everything

**Review by wallyworld — APPROVED** [review]
Just a couple of small things

File: src/charm.py (hunk 1 above)
Comment (alesstimec) [reply src/charm.py]: done

File: tests/test_charm.py (hunk 2 above)
Comment (alesstimec) [reply tests/test_charm.py:12]: done

**Pull request merged**